index.html

```diff
@@ -109,4 +109,11 @@ <h2>MJ (Marcos JSP) @MarcosJSP</h2>
     <h4>Key fingerprint=DBDD 73C0 73C7 22E2 A2B3 CAD3 F6D7 3AE3 7EBE 96D2</h4>
     <hr />
 
+    <h2>Todd @tvangundy</h2>
+    <a href="keys/tvangundy.asc">tvangundy.asc</a>
+    <a href="https://keys.openpgp.org/vks/v1/by-fingerprint/C07B19078BB6DA872C8A0015B173BE29FD0DF36D">0xb173be29fd0df36d</a>
```

Comment:
I uploaded the key to both openpgp and mitpgp servers previously.  

I able to encrypt/decrypt a file with the new key.

The key does show up on the openpgp server when I use the search url (http://keyserver.ubuntu.com/) 

and search on "c07b19078bb6da872c8a0015b173be29fd0df36d" 
or  searching on tvangundy@<all three email addresses>.

I'm not sure what to try next.  Suggestions welcome?  :)

Reply:
Yep, upload and "reverification" using the gpg command line worked.

I think the glitch was caused by me when I "kind of" created the key in two phases.  I created, uploaded and verified the key originally with only my tvangundy@horizenlabs.io email.  Then, later I added the other two emails IDs and uploaded again after that via the opengpg UI.  I think the correct thing to do is to use the gpg export command after making changes to the key so that verification can be accomplished again after the changes.